regression/contracts/loop-history-variables-01/main.c

```diff
@@ -0,0 +1,17 @@
+#include <assert.h>
+
+void main ()
+{
+  int *x, y, z; 
+
+  *x = &z;
+
+  while(y > 0)
+  __CPROVER_loop_invariant(*x == __CPROVER_loop_entry(*x))
```

Comment:
Could you also add a couple more loops to this file and test for struct members and scalars as well?